client/components/fields/editor/EventRelatedArticles/PreviewArticle.tsx

```diff
@@ -61,6 +61,13 @@ export class PreviewArticle extends React.PureComponent<IProps, IState> {
         );
         const headlineField = extractedFields['headline'];
         const bodyField = extractedFields['body_html'];
+        const subjectSchemes = appConfig?.SUBJECT_SCHEME_LABELS || {};
```

Comment:
I would rather avoid some extra config for this and rather show every CV we have stored on the instance, like we do in monitoring preview